core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -58,6 +58,9 @@
     @Nullable
     private final SSLSession sslSession;
 
+    @Nullable
+    private final ProxiedAddresses proxiedAddresses;
```

Comment:
Just wondering, do you think we can use this mechanism to handle `X-Forwarded-For` as well or would that be completely at the `Logger` layer?

Reply:
We wanna evolve this into something more generic so that it provides the access to the `X-Forwarded-For` headers. Just didn't expand that far in this PR. :-)

Reply:
Cool - IIUC, we don't have headers available yet at this point so thought it might be difficult, but we'll get to that later

Reply:
`HttpRequest` is given when `DefaultServiceRequestContext` is created, so I think it's trivial to get any request header.